recipes/stages/_base_/data/cls_semi_data.py

```diff
@@ -0,0 +1,30 @@
+_base_ = [
+    './data.py',
+    './pipelines/semisl_pipeline.py'
+]
+
+__dataset_type = 'ClsDirDataset'
+
+__train_pipeline = {{_base_.train_pipeline}}
+__test_pipeline = {{_base_.test_pipeline}}
+
+__samples_per_gpu = 32
+
+data = dict(
+    samples_per_gpu=__samples_per_gpu,
+    workers_per_gpu=2,
+    train=dict(
+        type=__dataset_type,
+        pipeline=__train_pipeline),
```

Comment:
Hmm, as far as I know, it should be revised to `train_pipeline_strong`. Am I right @harimkang?

Reply:
I just made a temp data cfg file to enable semi-recipe, because it was not importing(inheriting) any data cfg inside semi recipe file. I expect you to revise it when you start implementation.

Reply:
Okay, I'll do that :)